src/tintx/reader.py

```diff
@@ -248,7 +254,12 @@ def grids(self) -> Iterator[GridType]:
     @property
     def tracks(self) -> pd.DataFrame:
         """Pandas ``DataFrame`` representation of the tracked cells."""
-        return self._tracks
+        import geopandas as gpd  # type: ignore
+        from shapely import geometry  # type: ignore
```

Comment:
I think you want to add those imports to the top.

For the `# type: ignore` better add those to mypy.ini